share/completions/pygmentize.fish

```diff
@@ -0,0 +1,30 @@
+function __fish_print_pygmentize --argument-names type description
+    for line in (pygmentize -L $type | sed -e "s%^\* \(.*\):%\1%;tx;d;:x" | string split ", ")
```

Comment:
Well, that's easy - `string match -r '^\* \S+:' | string replace -r '^\* (\S+):' '$1'`. This would be another case where a "only-matching" option would be useful.

In this case, we can probably save that to a variable and use the following line as description.

Try this:

```fish
set -l lines (pygmentize -L $type | string match -r '^(?:\*|    ).*(?:)$' | string replace -r '\* (.*):$' '$1' | string trim)

while set -q lines[2]
    printf '%s\t%s\n' $lines[1] $lines[2]
    set -e lines[1]
    set -e lines[1]
end
```